src/components/permissions/permissions.vue

```diff
@@ -194,7 +194,8 @@ export default {
       border-top: 1px solid var(--lightest-gray);
     }
   }
-  /deep/ .cell {
+
+  >>> .cell {
```

Comment:
We actually had to revert this change you made earlier from `/deep/` to `>>>`, as vue-cli uses an older version of sass that doesn't support this yet...

```suggestion
  /deep/ .cell {
```